app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :users, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :posts, class_name: 'Post', foreign_key: 'post_id'
+
```

Comment:
- [ ] I noticed that you used plural names for post and user in the belongs_to association. This is not a good practice because it does not follow the Rails convention of naming associations. It also makes the code harder to read and understand, because plural names imply that a record belongs to many records, which is not true in this case. A comment belongs to one post and one user, not many posts and users.
I suggest that you use singular name for post and user in the belongs_to association. You can also omit the class_name and foreign_key options if they match the default values. For example, instead of writing this:

```rb

belongs_to :users, class_name: 'User', foreign_key: 'user_id'
belongs_to :posts, class_name: 'Post', foreign_key: 'post_id'

```

```rb
belongs_to :user
belongs_to :post
```

- [ ] [OPTIONAL] Also please consider using `after_save` for  #update_comments_counter instead of `after_create` here and make it a private method.

Example snippet:

```rb

class Comment < ApplicationRecord
  belongs_to :user
  belongs_to :post

  after_save :update_comments_counter

private

  def update_comments_counter
    post.increment!(:comments_counter)
  end
end
```